x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -216,7 +215,6 @@ const getCspStatus = async ({
       findings: findingsIndexStatusCspm,
       score: scoreIndexStatusCspm,
     },
-    installedPackagePoliciesTotalCspm,
```

Comment:
Why are we removing `installedPackagePoliciesTotalCspm` and  `installedPackagePoliciesTotalKspm`?

Reply:
installedCspPackagePolicies on CalculateCspStatus function is not being used
as such I removed it

Reply:
okay nice catch. Can we also remove `installedPackagePoliciesTotalVulnMgmt` in that case